app/controllers/hyrax/dashboard/nest_collections_controller.rb

```diff
@@ -18,6 +21,19 @@ def create_within
         end
       end
 
+      def process_nesting 
+        byebug
+         @parent_id = params[:parent_id]
+         @child_id = params[:child_id]
+         
+         parent_coll = Collection.find params[:parent_id]
+         child_coll = Collection.find params[:child_id]
+         parent_coll.add_member_objects ( params[:child_id] )
+   
+         #Not sure where to go from here.
+      end
```

Comment:
You will need to save the child for the indexing to make it into the solr doc.

@laritakr Is this what you would expect?  My testing in rails console required a save after setting member_of_collections.